doc/api/globals.md

```diff
@@ -598,30 +598,32 @@ This variable may appear to be global but is not. See [`module`][].
 
 <!-- YAML
 added: v21.0.0
+changes:
+  version: REPLACEME
+  pr-url: https://github.com/nodejs/node/pull/50412
+  description: The API is behind the CLI flag `--experimental-global-navigator`.
 -->
 
-> Stability: 1.1 - Active development
+> Stability: 1.1 - Active development. Use `--experimental-global-navigator` to
```

Comment:
Why is it considered as active development? There's no other public API we have to add in order to be spec compliant with WINTERCG?

Reply:
You approved https://github.com/nodejs/node/pull/50310 that set this status. It’s unrelated to this PR, feel free to propose a change in a different PR if you no longer think it applies here.

Reply:
The base global `navigator` is under active development in that we’re still adding properties to it. But individual properties like `userAgent` and `hardwareConcurrency` are arguably already stable, since they’re complete and we don’t plan to make any changes to them other than to address bugs.